capstone/src/main/java/com/google/step/clients/MetricsClient.java

```diff
@@ -128,9 +128,11 @@ public WeeklyPageView getCurrentPageViews(String restaurantKey) {
         // Map to maintain a reference to each restaurantList and make searching for them easy.
         // The map and the list share the reference to each restaurantPageViews.
         Map<String, List<WeeklyPageView>> restaurantNameMap = new HashMap<>();
```

Comment:
If you're doing a dual-map approach, would it make more sense to key the maps by the `restaurantKey` field, since you can guarantee uniqueness of that field for a single restaurant?

Reply:
The Id should also be unique.